DFC.FindACourseClient/Services/FakeHttpRequestSender.cs

```diff
@@ -0,0 +1,16 @@
+﻿using DFC.FindACourseClient.Contracts;
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Net.Http;
+
+namespace DFC.FindACourseClient.Services
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
Why do we need to exclude from coverage if only used by test project, shouldnt we move this to test csproj which will automatically get excluded for coverage?